conans/client/cmd/uploader.py

```diff
@@ -32,7 +31,8 @@ def check(self, upload_bundle, remote, force):
                 self._check_upstream_package(pref, prev_bundle, remote, force)
 
     def _check_upstream_recipe(self, ref, ref_bundle, remote, force):
-        self._output.info("Checking which revisions exist in the remote server")
+        output = ConanOutput(scope=str(ref))
+        output.info("Checking which revisions exist in the remote server")
```

Comment:
Do you think messages like this or "Preparing artifacts to upload" could be `highlight`? Similar to https://github.com/conan-io/conan/blob/02d0b7b6cb7697cd3b759a4a76aa11ac7f6283a0/conans/client/generators/__init__.py#L196

Reply:
I have actually added some "subtitle" level, differentiating the different stages of the upload process, I think it looks good separating a bit the actions:
```bash
-------- Checking server existing packages --------
hello0/1.2.1@frodo/stable: Checking which revisions exist in the remote server
hello1/1.2.1@frodo/stable: Checking which revisions exist in the remote server
hello2/1.2.1@frodo/stable: Checking which revisions exist in the remote server

-------- Preparing artifacts for upload --------
Please log in to "default" to perform this action. Execute "conan remote login" command.
Remote 'default' username: Please enter a password for "admin" account: 
-------- Uploading artifacts --------
Uploading recipe 'hello0/1.2.1@frodo/stable#761f54e34d59deb172d6078add7050a7'
Uploading recipe 'hello1/1.2.1@frodo/stable#b8c4bba452ad3c38a715b19957c60729'
Uploading recipe 'hello2/1.2.1@frodo/stable#b3b020bf824a1ff93b50d9061dcb9dbf'
Upload complete

Upload summary:
default
  hello0/1.2.1@frodo/stable
    revisions
      761f54e34d59deb172d6078add7050a7 (Uploaded)
        packages
  hello1/1.2.1@frodo/stable
    revisions
      b8c4bba452ad3c38a715b19957c60729 (Uploaded)
        packages
  hello2/1.2.1@frodo/stable
    revisions
      b3b020bf824a1ff93b50d9061dcb9dbf (Uploaded)
        packages`
```